backend/activity_log/tests/test_api_views.py

```diff
@@ -68,15 +68,12 @@ class TestActivityLogEventTypeViewSet(APITestCase, APITestCaseMixin):
     def setUp(self):
         self.today = timezone.now()
 
-        self.entry1 = factories.EventTypeFactory()
```

Comment:
I suppose I'm a bit confused by which of these you deleted....since the ticket is on creating the api for an event_type, would you not test it by creating one of those instead of a log entry - but I'm not sure how much difference this makes. Maybe we can ask Sheena later....